src/Nokia_LCD.cpp

```diff
@@ -196,9 +224,22 @@ bool Nokia_LCD::send(const unsigned char lcd_byte, const bool is_data) {
     digitalWrite(kDc_pin, is_data);
 
     // Send the byte
+    if (kUsingHardwareSPI) {
+        SPI.begin();
+        SPI.setClockDivider(SPI_CLOCK_DIV4);
+        SPI.setDataMode(SPI_MODE0);
+        SPI.setBitOrder(MSBFIRST);
+    }
     digitalWrite(kCe_pin, LOW);
```

Comment:
Does the `SPI.begin()` etc need to happen before pulling the chip select pin low?
I am wondering if we could combine these two `if (kUsingHardwareSPI)` statements on L227 and L234.

Reply:
As I have found in most examples, the CS pin is going LOW before SPI.begin();
But I tried to call the method before setting the pin to LOW and it works.